testsuite/scripts/samples/test_electrophoresis.py

```diff
@@ -20,29 +20,29 @@
 import numpy as np
 
 sample, skipIfMissingFeatures = importlib_wrapper.configure_and_import(
-    "@SAMPLES_DIR@/electrophoresis.py")
+    "@SAMPLES_DIR@/electrophoresis.py", N_SAMPLES=400)
 
 
 @skipIfMissingFeatures
 class Sample(ut.TestCase):
     system = sample.system
 
-    @ut.skipIf(tuple(map(int, np.__version__.split("."))) < (1, 10),
-               "not supported for numpy < v1.10")
     def test_persistence_length(self):
-        value = np.mean(sample.fit[0])
-        self.assertTrue(5. < value < 110., "length = {:.0f}".format(value))
+        self.assertAlmostEqual(sample.persistence_length, 28., delta=2,
+                               msg="persistence length of {:.1f} not correct".format(sample.persistence_length_obs))
```

Comment:
```suggestion
                               msg="persistence length of {:.1f} not correct".format(sample.persistence_length))
```
I don't think the `msg` argument was designed with this kind of code duplication in mind, which is prone to subtle copy-paste errors that will actually hurt maintainers instead of guiding them. The default error message of unittest assertions is already clear enough, showing both the function name (`test_persistence_length`) and the faulty variable if the assertion is a one-liner. I would vote for removing custom messages that only format the variable names. I'm not even sure it can be automated via `inspect` in a reliable way (see https://stackoverflow.com/a/2749881, https://stackoverflow.com/a/2749857).

Reply:
I don't have a strong opinion on that  but I also think that we should not over-engineer one-liners.